strings-csharp/SubstringInString/SubstringInString/Benchmarking.cs

```diff
@@ -1,26 +1,30 @@
 ﻿using BenchmarkDotNet.Attributes;
 using System.Text.RegularExpressions;
 
+[Config(typeof(AntiVirusFriendlyConfig))]
 [MemoryDiagnoser]
 [Orderer(BenchmarkDotNet.Order.SummaryOrderPolicy.FastestToSlowest)]
 public class Benchmarking
 {
-    private const string str = @"Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    private const string str = """
+                               Lorem ipsum dolor sit amet, consectetur adipiscing elit.
                                Nullam quis lorem nec lorem imperdiet tempus. Phasellus lorem justo,
                                consequat ut lorem sit amet, tempor feugiat lorem.
                                Proin lorem velit, aliquam vel lorem in, consequat lorem nulla.
                                Sed lorem justo, eleifend eu lorem id, consectetur lorem neque.
                                Vestibulum lorem mauris, placerat sit amet lorem nec, laoreet lorem nisi.
-                               Sed nec lorem libero. Nullam lorem lorem, ullamcorper quis lorem vitae.";
+                               Sed nec lorem libero. Nullam lorem lorem, ullamcorper quis lorem vitae.
+                               """;
 
     public string toFind = "Lorem";
     private Regex regex;
+    string pattern = @"ip";
 
     [GlobalSetup]
     public void Setup()
     {
         // Compile the regex pattern into a Regex object using the Compiled option
-        regex = new Regex(toFind, RegexOptions.Compiled);
+        regex = new Regex(pattern, RegexOptions.Compiled);
```

Comment:
@lynda-nneka the Regex search should match the other searches. Since this is a benchmark where we are making a comparison amongst different methods for achieving the same goal, they each need to return the same results.